lib/rules/require-expect.js

```diff
@@ -28,6 +28,15 @@ module.exports = {
         schema: [
             {
                 "enum": ["always", "except-simple", "never", "never-except-zero"]
+            },
+            {
+                "type": "object",
+                "properties": {
+                    "allowFunctions": {
+                        "type": "array"
```

Comment:
Can we add to this to require that all items in the array should be strings? Also, can we only allow this option if the previous value is except-simple?